README.md

```diff
@@ -1,5 +1,8 @@
 teambot
 =======
+
+[![Build Status](https://travis-ci.org/RyanBalfanz/teambot.svg?branch=settings-module-config)](https://travis-ci.org/RyanBalfanz/teambot)
```

Comment:
Haha, yeah I was on Travis and I saw it start. I was wondering how that happened… until I came back here. 👍 

<img width="769" alt="screen shot 2017-03-29 at 9 33 37 pm" src="https://cloud.githubusercontent.com/assets/133278/24487836/86b4d2d2-14c7-11e7-977f-5ecfc4603772.png">